parmed/openmm/parameters.py

```diff
@@ -214,6 +222,19 @@ def _find_unused_types(self, skip_residues):
                     keep_types.add(atom.type)
         return {typ for typ in self.atom_types if typ not in keep_types}
 
+    def _find_identical_residues(self):
+        # this is meant for monoatomic ions ONLY
+        skip_ident_residues = set()
+        monoatomic_types = set()
+        for name, residue in iteritems(self.residues):
+            if len(residue.atoms) == 1:
+                typ = residue.atoms[0].type
```

Comment:
Should we add `type` or `atomic_number` to this set?  Since OpenMM does element matching to find duplicate residues, I think the latter is a better choice here.  What do you think?